packages/js/src/preferences/preference.ts

```diff
@@ -42,15 +43,19 @@ export class Preference {
     this.workflow = preference.workflow;
   }
 
-  update({ channelPreferences }: { channelPreferences: ChannelPreference }): Result<Preference> {
+  update({
+    channels,
+    // @deprecated use channels instead
+    channelPreferences,
```

Comment:
We are already doing a breaking change for the apiUrl, so I think we can just continue 😅 Just need to make sure to tell about this on release notes and changelog

Reply:
Oh nvm, just noticed it was on the httpclient only

Reply:
We are not doing a breaking change as we have deprecated the old `backendUrl` and introduced the `apiUrl` on the `NovuOptions`. Similarly, on the update preferences args.